guillotina/contrib/pubsub/utility.py

```diff
@@ -0,0 +1,106 @@
+from guillotina.profile import profilable
+from guillotina.utils import resolve_dotted_name
+from guillotina.contrib.pubsub.exceptions import NoPubSubDriver
+from typing import Any
+from typing import Callable
+import pickle
+import asyncio
+import logging
+
+app_settings = {
```

Comment:
Why are these app_settings here? Shouldn't it be loading from the app_settings in parent module?